src/attestation/azext_attestation/generated/_client_factory.py

```diff
@@ -9,11 +9,37 @@
 # --------------------------------------------------------------------------
 
 
-def cf_attestation(cli_ctx, *_):
+def cf_attestation_mgmt(cli_ctx, *_):
     from azure.cli.core.commands.client_factory import get_mgmt_service_client
     from ..vendored_sdks.azure_mgmt_attestation import AttestationManagementClient
     return get_mgmt_service_client(cli_ctx, AttestationManagementClient)
 
 
 def cf_attestation_provider(cli_ctx, *_):
-    return cf_attestation(cli_ctx).attestation_provider
+    return cf_attestation_mgmt(cli_ctx).attestation_provider
+
+
+def cf_attestation_data(cli_ctx, *_):
+    from ..vendored_sdks.azure_attestation import AttestationClient
+    from azure.cli.core._profile import Profile
+
+    profile = Profile(cli_ctx=cli_ctx)
+    cred, _, _ = profile.get_login_credentials(
+        resource="https://attest.azure.net")
```

Comment:
will the url be different in different clouds?

Reply:
@yungezz Good question. I think I need to confirm this with the service team.

Reply:
> @yungezz Good question. I think I need to confirm this with the service team.

@bim-msft Is this resolved?

Reply:
@YalinLi0312 service need to register endpoints to different clouds. which will be in a separate PR.